myapp.rb

```diff
@@ -6,6 +6,7 @@
 require "./models/message"
 require "./models/user"
 require "./models/follower"
+require "./models/latest"
```

Comment:
What do we use the `latest` model for in our app?

Also; Do we need to require each model indepedently? I found [this stackoverflow post](https://stackoverflow.com/questions/735073/best-way-to-require-all-files-from-a-directory-in-ruby) but am uncertain based on their discussions if it actually is possible or not for our setup etc.

Reply:
See newest commit